django/http/multipartparser.py

```diff
@@ -116,6 +118,8 @@ def parse(self):
 
         encoding = self._encoding
         handlers = self._upload_handlers
+        # Number of bytes that have been read.
+        data_size = 0
```

Comment:
Having this around L152 would be very marginally cleaner from my POV. (Ie. it's similar to `num_post_keys` so makes sense for it to get initialized at same point in the codebase)